### ISSUE TITLE
Not adding to my HA

### ISSUE THREAD

**Issue by Ties1208** [ualex73/slide#7]
Hello,

I added this intergration to my HA. But my Slide is not popping up in the devices. how is this possible?

**Comment by ualex73** [comment]
Where you able to fix it?

**Comment by Ties1208** [comment]
no.. is used it all the time trough google home. but now that slide is closing the doors i cant use it anymore.. if added the local code with the password and the right ip adres in my yaml. but it isnt working.

**Comment by ualex73** [comment]
What type of error are you getting in the home-assistant.log? Also, which version of the slide GitHub are you running (1.0/1.1/1.2/1.3)?

**Comment by Ties1208** [comment]
Exactly the same errors...

**Comment by Ties1208** [comment]
<img width="260" alt="Slide" src="https://github.com/ualex73/slide/assets/116915611/f81711a0-fe9d-44b6-baf8-ad1a0c3f9248">
 This is what i got in my configuration file.. am i doing something wrong?

**Comment by ualex73** [comment]
Change api_version: 2 to api_version: 1 and try again. Btw the password is not that interesting anymore without the cloud.

**Comment by Ties1208** [comment]
Well... that work bro! thank you so much for helping! btw i tried without password but that isnt working. he needs the device code.
![Afbeelding van WhatsApp op 2024-05-24 om 10 24 44_a5ec889a](https://github.com/ualex73/slide/assets/116915611/4bc98d95-b99b-4a0d-ae91-7d7d66e3d421)

**Comment by ualex73** [comment]
Good it works now with ap_version=1, but the password error is strange... I will try to reproduce it.

**Comment by wesaidso** [comment]
I read that only from ap_version=2 the password is no longer required. Just ordered a few slides and playing around with it. Thanks for this custom plugin, integrates easily in HA!

**Comment by sandervandegeijn** [comment]
Afraid I'm having the same problem. Running home assistant 2024.6.2 in docker, I have a slide with fw 2.0 according to the slide of Kaj

Config:

<img width="534" alt="image" src="https://github.com/ualex73/slide/assets/2643715/0c02ad52-ecdb-4304-a40f-5223db37e741">

Custom components dir contents:
<img width="506" alt="image" src="https://github.com/ualex73/slide/assets/2643715/338a9940-bb62-49e5-a029-bfff8a9a0608">


This is from inside the home assistant docker container: 

```
5124435a4cab:/config# curl -X POST http://192.168.3.10/rpc/Slide.SetPos -H "Content-Type: application/json" -d '{"pos": 0}'
{"response":"success"}
5124435a4cab:/config# curl -X POST http://192.168.3.10/rpc/Slide.SetPos -H "Content-Type: application/json" -d '{"pos": 1}'
{"response":"success"}
```

So I have connectivity, ip address is correct and curtains are responding to the commands.

HA:

<img width="1458" alt="image" src="https://github.com/ualex73/slide/assets/2643715/7049d32b-0d40-480e-b772-e56afd6b6c83">

<img width="1021" alt="image" src="https://github.com/ualex73/slide/assets/2643715/d4e9f497-c1cb-4378-bc73-24d3546c2f2b">

**Comment by ualex73** [comment]
I will update the README, because the custom component Slide does not pop up under "Devices". You can find it in "Developer tools" -> "States".

**Comment by sandervandegeijn** [comment]
You are right! Thanks!

Two things threw me off:

1. There is still a Slide under devices, but that would be the original implementation with the same name. This is a bit confusing
2. I did search under states, but as slide. Didn't think to search for cover. Kind of stupid, still, the slide name in the device name would be helpful.

**Comment by ualex73** [comment]
It is added to the README now.
